### ISSUE TITLE
Title bar still remains white

### ISSUE THREAD

**Issue by Noodles-321** [Rosmaninho/Zotero-Dark-Theme#4]
First of all, thank you so much for this dark theme! This is really an eye's cream for me!

I found the title bar is still white, which is the only difference compared with your screenshot. But I am not sure if this is something caused by the Windows settings. I am already using a system-wide dark mode on my Window 10, and both my Firefox and file explorer have dark title bars.

Have you by any chance experienced this?

![Snipaste_2021-02-06_22-05-06](https://user-images.githubusercontent.com/37469405/107120823-17635380-68ca-11eb-9af1-ab6eaac19f14.png)

**Comment by Rosmaninho** [comment]
Ah yes! The window borders are handled by Windows, not Zotero. 
You can change that in the personalize section of the Windows settings (right-click desktop -> Personalize -> Colors -> check Show accent color on the following surfaces Title bars and window borders).

I made my window borders fully black by changing it in the Windows registry. 
https://www.tenforums.com/customization/115714-how-set-custom-active-title-bar-color-via-registry.html

You can also use this program:
https://www.tenforums.com/software-apps/56236-s-m-c-c-superflys-mouse-assisted-color-chooser.html

**Comment by Noodles-321** [comment]
Thank you for the links. It works. 

I am sorry the following is a bit off-topic...
I did not manage to remove the one-pixel blue border on active windows appeared after the change. (It is also mentioned in [the last comment in your first link](https://www.tenforums.com/customization/115714-how-set-custom-active-title-bar-color-via-registry.html#post1433256).) I googled around but got no luck. May I ask how you managed to remove it?
![Snipaste_2021-02-07_22-17-37](https://user-images.githubusercontent.com/37469405/107150139-4dbad480-6997-11eb-833f-86a60bba3a91.png)
